tapo/examples/tapo_h100.rs

```diff
@@ -113,5 +115,27 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     }
 
+    let supported_ringtones = hub.get_supported_alarm_type_list().await?;
+    info!("Supported ringtones: {:?}", supported_ringtones);
```

Comment:
Is this necessary in the example, given that it's an edge and/or advanced use case?

Reply:
Removed